documentation/docs/specs/did/iota_did_method_spec.md

```diff
@@ -58,246 +86,90 @@ base-char = 1-9 A-H J-N P-Z a-k m-z
 
 ### IOTA-Network
 
+**TODO should be replaced by shimmer? **
+
 The iota-network is an identifier of the network where the DID is stored. This network must be an IOTA Tangle, but can either be a public or private network, permissionless or permissioned.
 
 The following values are reserved and cannot reference other networks:
+
 1. `main` references the main network which refers to the Tangle known to host the IOTA cryptocurrency.
 2. `dev` references the development network known as "devnet" maintained by the IOTA Foundation.
 
 When no IOTA network is specified, it is assumed that the DID is located on the `main` network. This means that the following DIDs will resolve to the same DID Document:
+
 ```
-did:iota:main:H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV
-did:iota:H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV
+did:iota:main:0xe4edef97da1257e83cbeb49159cfdd2da6ac971ac447f233f8439cf29376ebfe
+did:iota:0xe4edef97da1257e83cbeb49159cfdd2da6ac971ac447f233f8439cf29376ebfe
 ```
 
 ### IOTA-Tag
 
-The IOTA tag references an indexation which resolves to the initial DID Messages. 
-
-#### Generation
-
-The following steps MUST be taken to generate a valid tag:
-* Generate an asymmetric keypair using a supported verification method type.
-* Hash the public key using `BLAKE2b-256` then encode it using [Base58-BTC](https://tools.ietf.org/id/draft-msporny-base58-01.html).
-
-This public key MUST be embedded into the DID Document (see [CRUD: Create](#create)).
+The IOTA tag references an `Alias ID` identitfying the Alias Output where the DID Document is stored.
+The tag will not be known before the generation of the DID. It will be assigned when the Alias Output is created.
 
-## DID Messages
+### Anatomy of serialized document
 
-DID Documents associated with the `did:iota` method consist of a chain of data messages, called "DID messages", published to a Tangle. The Tangle has no understanding of DID messages and acts purely as an immutable database. The chain of DID messages and the resulting DID Document must therefore be validated on the client side. 
+In the `Metadata` of the Alias Output must be a JSON document containing two properties:
```

Comment:
This is incorrect, it's a byte-packed struct with header fields, like the DID Message. It's also `stateMetadata`, not `metadata`, which is a separate feature.

`DID<VERSION><ENCODING><PAYLOAD>`

This should conveyed by a Markdown table similar to the output specifications, e.g. https://hackmd.io/SULTqAt5Q7yQwWTghJtZCw#DID-Output